go/runtime/host/host.go

```diff
@@ -53,7 +53,7 @@ type Runtime interface {
 	Start() error
 
 	// Restart attempts to restart a runtime so that it will be ready to service new requests.
-	Restart(ctx context.Context) error
+	Restart(ctx context.Context, force bool) error
```

Comment:
Would it make more sense to rename this to `Abort` as now it does not actually do a `Restart` unless it's necessary? (or if the `force` flag is on)

Reply:
Yeah good idea, and mention in the documentation comment in which case the runtime is restarted (e.g., if the abort fails or the force flag is set).